targets/gtk-extra

```diff
@@ -8,12 +8,23 @@ DESCRIPTION='GTK-based tools including gdebi, gksu, and a simple browser.'
 
 ### Append to prepare.sh:
 install_dummy network-manager network-manager-gnome
-install gdebi gksu netsurf-gtk
+install gdebi gksu
 
-# Add netsurf-gtk to Debian-alternatives if not already there
-netsurf='/usr/bin/netsurf-gtk'
-for link in x-www-browser gnome-www-browser; do
-    if ! update-alternatives --query "$link" | grep -q "$netsurf"; then
-        update-alternatives --install "/usr/bin/$link" "$link" "$netsurf" 10
+# if netsurf-gtk is not available (in certain debian and kali releases), fallback to dillow.
+if install netsurf-gtk; then
+    netsurf='/usr/bin/netsurf-gtk'
+    for link in x-www-browser gnome-www-browser; do
+        if ! update-alternatives --query "$link" | grep -q "$netsurf"; then
+            update-alternatives --install "/usr/bin/$link" "$link" "$netsurf" 10
+        fi
+    done
+else
+    if install dillo; then
+       dillo='/usr/bin/dillo'
+        for link in x-www-browser gnome-www-browser; do
```

Comment:
is this needed for the dillo install (i.e., it doesn't do it automatically)?  If so, please restructure the code to avoid duplication.